packages/components/src/Slider/Slider.component.js

```diff
@@ -254,7 +280,16 @@ class Slider extends React.Component {
 						min={min}
 						max={max}
 						handle={noValue ? undefined : this.state.handle}
-						className={classnames(theme['tc-slider-rc-slider'], 'tc-slider-rc-slider')}
+						className={classnames(
+							theme['tc-slider-rc-slider'],
+							{[theme['tc-slider-rc-slider--track-equals']]: modeStyles === 'equals'},
```

Comment:
You can use a function instead to determinate the right class

Reply:
We won't do the integrity check for mode type now so classes can be added inline by checking the value of `mode` in the normal way with classnames.